bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/device/CommandHandler.java

```diff
@@ -114,9 +114,9 @@ protected String nm() {
 
     protected int getMaxLightLevel(InsteonChannelConfiguration conf, int defaultLevel) {
         Map<String, @Nullable String> params = conf.getParameters();
-        if (conf.getFeature().contains("dimmer") && params.containsKey("dimmermax")) {
+        String dimmerMax;
+        if (conf.getFeature().contains("dimmer") && (dimmerMax = params.get("dimmermax")) != null) {
```

Comment:
I'd try to avoid inline variable assignment if possible.
```suggestion
        String dimmerMax = params.get("dimmermax");
        if (conf.getFeature().contains("dimmer") && dimmerMax != null) {
```